rest-api/src/main/resources/paths/studies/v5_studies_studyId_participants_userId_adherence_weekly.yml

```diff
@@ -1,5 +1,5 @@
 get:
```

Comment:
Just something I came across, the path variable in the server is called "userIdToken" instead of "userId".

Reply:
I think that's okay. We user "userId" throughout the SDK and should be documenting somewhere that this can be a value like "healthcode:yyy"